cleanlab/experimental/tf_nlp.py

```diff
@@ -0,0 +1,129 @@
+"""
+A Scikeras classifier (adapted for huggingface models) which
+can be used for finding label issues in any text datasets.
+"""
+
+from typing import Dict
+from scikeras.wrappers import KerasClassifier
+from sklearn.preprocessing import FunctionTransformer
+import tensorflow as tf
+import numpy as np
+
+
+class HuggingfaceKerasClassifier(KerasClassifier):
+    def __init__(self, train_input: Dict, seq_len: int, **kwargs):
+        """
+        Basic scikeras/scikeras does not directly provide the option of having
+        a multi-input model, i.e the input must be in the form (num_samples, feature_size). However, you can work around this problem by
+        adding the `feature_encoder` property to the class which extends
+        Scikitlearn's BaseEstimator.
+
+        Example of use:
+
+        * ``model = HuggingfaceKerasClassifier(``
+        * ``    # --- model function parameters ---``
+        * ``    model=model_fn,``
+        * ``    n_classes=2,``
+        * ``    # --- HuggingfaceKerasClassifier Parameters ---``
+        * ``    train_input=dict(train_input),``
+        * ``    seq_len=20,``
+        * ``    #   --- TF training Parameters ---``
+        * ``    optimizer=tf.keras.optimizers.Adam(2e-5),``
+        * ``    loss=tf.keras.losses.BinaryCrossentropy(),``
+        * ``    metrics=['accuracy'],``
+        * ``    epochs=10,``
+        * ``    batch_size=64,``
+        * ``    shuffle=True,``
+        * ``    callbacks=[early_stopping],``
+        * ``    verbose=True``
+        * ``)``
+        * `` ``
+        * ``lnl = CleanLearning(clf=model, cv_n_folds=3)``
+        * ``lnl.fit(training_ids, train_labels, clf_kwargs={'validation_data': val_dataset})``
+        * ``lnl.score(test_ids, test_labels)``
```

Comment:
I don't think `lnl.score()` is actually computing the correct thing right now, due to the problem I pointed out below for how `predict()` is implemented?

Reply:
Honestly, I don't know why there is that line, I think comes from my personal code because it really doesn't make sense in our case. I will replace it with an example of the predict function if you agree.